.pre-commit-config.yaml

```diff
@@ -21,6 +21,7 @@ repos:
     rev: 24.4.2
     hooks:
       - id: black
+        exclude: ./doc/source/conf\.py
```

Comment:
Pre-commit runs on this file, same formatting in the pyproject.toml. Issue was you cannot pass the arg to black, but instead have to prevent pre-commit from passing it to black.